projects/demo/src/modules/components/legend-item/examples/2/index.ts

```diff
@@ -1,17 +1,34 @@
+import {AsyncPipe, NgForOf} from '@angular/common';
 import {Component, inject} from '@angular/core';
 import {changeDetection} from '@demo/emulate/change-detection';
 import {encapsulation} from '@demo/emulate/encapsulation';
+import {TuiLegendItemComponent, TuiRingChartComponent} from '@taiga-ui/addon-charts';
+import {TuiAmountPipe} from '@taiga-ui/addon-commerce';
 import {tuiPure, tuiSum} from '@taiga-ui/cdk';
-import {TuiAlertService, tuiFormatNumber} from '@taiga-ui/core';
+import {
+    TuiAlertService,
+    tuiFormatNumber,
+    TuiNotificationComponent,
+    TuiSvgComponent,
+} from '@taiga-ui/core';
 
 @Component({
-    selector: 'tui-legend-item-example-2',
+    standalone: true,
+    imports: [
```

Comment:
Add checkbox import to fix screenshots

![image](https://github.com/taiga-family/taiga-ui/assets/35179038/63012b8b-b609-418c-8e42-dfd1c87c8019)

Reply:
thank you

Reply:
fixed